packages/lesswrong/lib/collections/posts/views.js

```diff
@@ -593,12 +593,16 @@ Posts.addView("afRecentDiscussionThreadsList", terms => {
       af: true,
     },
     options: {
-      sort: {lastCommentedAt:-1},
+      sort: {afLastCommentedAt:-1},
       limit: terms.limit || 12,
     }
   }
 })
-// Covered by the same index as `recentDiscussionThreadsList`
+ensureIndex(Posts,
+  augmentForDefaultView({ afLastCommentedAt:-1, baseScore:1, hideFrontpageComments:1 }),
```

Comment:
Would be better with `hideFrontpageComments` in front, since it's boolean, but this index will work well enough.

```suggestion
  augmentForDefaultView({ hideFrontpageComments:1, afLastCommentedAt:-1, baseScore:1 }),
```